ElementX/Sources/Other/Extensions/String.swift

```
@@ -51,4 +51,15 @@ extension String {
         }
         return abs(hash)
     }
+    
+    func asciified() -> String {
+        guard !canBeConverted(to: .ascii) else {
+            return self
```

This is potentially unsafe if it silently fails converiting to ascii. It would be safe to return a nullable and let the user handle it.

---

Oh, excellent point! 👍

---

Interestingly `"someascii".canBeConverted(to: .ascii)` is `false` so I had to add another check to see if the string is already in ASCII.